VirtualList/useThemeVirtualList.js

```diff
@@ -49,6 +49,10 @@ const useSpottable = (props, instances) => {
 
 	useSpotlightConfig(props, {spottable: mutableRef});
 
+	if (snapToCenter) {
+		SpotlightAccelerator.setFrequency([7]);
```

Comment:
When `snapToCenter` becomes a falsy value, the frequency value is not recovered to the initial value.

Reply:
sure